operator/src/main/resources/application.properties

```diff
@@ -14,3 +14,6 @@ quarkus.openshift.env.vars.operator-keycloak-image=${operator.keycloak.image}
 # Bundle config
 quarkus.operator-sdk.bundle.package-name=keycloak-operator
 quarkus.operator-sdk.bundle.channels=fast
+
+quarkus.operator-sdk.controllers.keycloakcontroller.namespaces=${quarkus.operator-sdk.namespaces:JOSDK_WATCH_CURRENT}
```

Comment:
This shouldn't be needed: `quarkus.operator-sdk.namespaces` is the value that should be used by default if a reconciler doesn't provide its own value.